scene/debugger/scene_debugger.cpp

```diff
@@ -1530,21 +1523,9 @@ void RuntimeNodeSelect::_click_point() {
 		message.append(items[0].item->get_instance_id());
 		EngineDebugger::get_singleton()->send_message("remote_node_clicked", message);
 	} else if (list_shortcut_pressed || node_select_mode == SELECT_MODE_LIST) {
-		if (!selection_list->is_inside_tree()) {
-			root->add_child(selection_list);
+		if (!selection_list) {
+			_open_selection_list(items, pos);
 		}
-
-		selection_list->clear();
-		for (const SelectResult &I : items) {
-			selection_list->add_item(I.item->get_name());
-			selection_list->set_item_metadata(-1, I.item);
-		}
```

Comment:
@akien-mga 
> How do you evaluate the cost of re-creating this list every time it's needed?

The cost of this PR's change differs only in that it re-creates the `PopupMenu` itself. The process of populating the list remains unchanged from the current implementation.